procedures/release-duty/index.rst

```diff
@@ -63,24 +65,23 @@ Email
 
-*  All types of sign-offs and approvals should go to `release signoff mailing list <https://mail.mozilla.org/listinfo/release-signoff>`__
-*  All releng automation emails should go to `release-automation-notifications <https://groups.google.com/a/mozilla.com/forum/?hl=en#!forum/release-automation-notifications>`__
-*  All discussion topics should go to `release drivers mailing list <https://mail.mozilla.org/listinfo/release-drivers>`__
+ * All types of sign-offs and approvals should go to `release signoff mailing list <https://mail.mozilla.org/listinfo/release-signoff>`__
+ * All discussion topics should go to `release drivers mailing list <https://mail.mozilla.org/listinfo/release-drivers>`__
 
 These other mailing lists will likely have useful discussions and information:
-* `RelEng internal mailing list <release@mozilla.com>`__
-* `Thunderbird release drivers mailing list <https://mail.mozilla.org/listinfo/thunderbird-drivers>`__
-* `release-automation-notifications-thunderbird mailing list <https://mail.mozilla.org/listinfo/release-automation-notifications-thunderbird>`__
-* (optional) `release-automation-notifications-dev mailing list <https://groups.google.com/a/mozilla.com/forum/#!forum/release-automation-notifications-dev>`__
+ * `RelEng internal mailing list <release@mozilla.com>`__
+ * `Thunderbird release drivers mailing list <https://mail.mozilla.org/listinfo/thunderbird-drivers>`__
+ * `release-automation-notifications-thunderbird mailing list <https://mail.mozilla.org/listinfo/release-automation-notifications-thunderbird>`__
+ * (optional) `release-automation-notifications-dev mailing list <https://groups.google.com/a/mozilla.com/forum/#!forum/release-automation-notifications-dev>`__
 
 Meetings and Calendars
 ~~~~~~~~~~~~~~~~~~~~~~
 
 Regular meetings are a vital part of making sure all the teams are kept informed and consulted during the release
 process. To view those meetings in your calendar you need to subscribe/be added to the following calendars:
 
-* `Public- Release Engineering <https://calendar.google.com/calendar/embed?src=mozilla.com_2d32343333353036312d393737%40resource.calendar.google.com>`__ (so that you attend the weekly post-mortem meeting)
-* `Releases Scheduling <https://calendar.google.com/calendar/embed?src=mozilla.com_dbq84anr9i8tcnmhabatstv5co@group.calendar.google.com>`__ (so that you can attend the Tuesday/Thursday channel meetings. You can add it following RelMan's `docs <https://wiki.mozilla.org/Release_Management#Calendar_Updating>`__) – If their instructions don't work, try to the “Add to Google Calendar” button at the `web version of the calendar <https://calendar.google.com/calendar/embed?src=mozilla.com_dbq84anr9i8tcnmhabatstv5co@group.calendar.google.com>`__.
+ * `Public- Release Engineering <https://calendar.google.com/calendar/embed?src=mozilla.com_2d32343333353036312d393737%40resource.calendar.google.com>`__ (so that you attend the weekly post-mortem meeting)
```

Comment:
We no longer have a formal post-mortem, so maybe we should remove this from now?